doc/bc/changes-6.0.md

```diff
@@ -211,6 +211,13 @@ Changes affecting version compatibility with former or future versions.
   Those need to be changed to custom content view controllers, that use a contentId instead of a locationId as an
   argument. The location is available in the `$parameters` array.
 
+* The `viewBlock` and `viewBlockById` `PageController` actions are deprecated.
+  `viewAction` must be used instead. It accepts both `block` and `id` as parameters.
```

Comment:
`viewAction` and `embedAction` docs mention location object as an extra param too :)

I would mention them, I see no reason why not. If you're thinking what I'm thinking, the bug was actually inability to use `render_esi` with `ez_page:viewBlock` as it only accepted block objects and not block IDs. That was solved by adding `viewBlockById` method in `ez_page` controller.